x-pack/plugins/ml/public/application/components/data_grid/data_grid.tsx

```diff
@@ -27,6 +28,8 @@ import { INDEX_STATUS } from '../../data_frame_analytics/common';
 
 import { euiDataGridStyle, euiDataGridToolbarSettings } from './common';
 import { UseIndexDataReturnType } from './types';
+// TODO Fix row hovering + bar highlighting
+// import { hoveredRow$ } from './column_chart';
```

Comment:
commented code 👀

Reply:
This will be solved in a follow up. Originally this code was used to work with `EuiBasicTable` which had a callback when hovering table rows. `EuiDataGrid` doesn't come with that out of the box so it needs more investigation.

Reply:
Sure it's fine to resolve in a follow-up, but with VCS there is no need in keeping the commented code 🙂